coalib/coala.py

```diff
@@ -34,9 +30,11 @@ def main():
         args = default_arg_parser().parse_args()
 
         if args.show_bears:
+            from coalib.settings.ConfigurationGathering import get_filtered_bears
             local_bears, global_bears = get_filtered_bears(
                 args.filter_by_language, log_printer)
 
+            from coalib.output.ConsoleInteraction import show_bears
```

Comment:
functools is probably not that relevent.

The coala imports are very evil because they again have lots and lots of imports and we're basically spending all the time on importing stuff